### PR TITLE
fix(unified-header): fixed keyboard event on the left nav link

### PR DESCRIPTION
**Pull request by marijohannessen**
Resolves https://github.ibm.com/Bluemix/carbon-issues/issues/282

Added a conditional that programmatically triggers a link click when the left nav item is selected with a keyboard.

### CODE REVIEW

**Comment by iangfleming** [comment]
👍 ✅  works great!

closes https://github.ibm.com/Bluemix/carbon-issues/issues/282

**Comment by tw15egan** [comment]
👍 ✅

File: src/components/unified-header/left-nav.js [hunk 1]
```diff
@@ -220,6 +220,7 @@ class LeftNav extends mixin(createComponent, initComponentBySearch) {
             const childItem = eventMatches(evt, this.options.selectorLeftNavNestedListItem);
             const hasChildren = eventMatches(evt, this.options.selectorLeftNavListItemHasChildren);
             const flyoutItem = eventMatches(evt, this.options.selectorLeftNavFlyoutItem);
+            const hasLinkItem = !(leftNavItem.querySelector(this.options.selectorLeftNavListItemLink) === undefined);
```
Comment (asudoh) [inline src/components/unified-header/left-nav.js:223]: `querySelector()` returns `null` if no matching element is found, so strict equal check against `undefined` would never be true.

Comment (hellobrian) [reply src/components/unified-header/left-nav.js:223]: @marijohannessen ☝️

**Pull request merged**